### PR TITLE
[Constants] Add constant to enable Ngrok

### PR DESCRIPTION
**Pull request by valouvaliavlo**

### CODE REVIEW

File: octobot_services/constants.py [hunk 1]
```diff
@@ -65,6 +65,7 @@
 
 # Webhook
 CONFIG_WEBHOOK = "webhook"
+CONFIG_ENBALE_NGROK = True
```
Comment (Herklos) [inline octobot_services/constants.py:68]:
```suggestion
CONFIG_ENABLE_NGROK = True
```

Comment (valouvaliavlo) [reply octobot_services/constants.py:68]: It's fix

**Comment by Herklos** [comment]
Can you also add the two other variables please ?

File: octobot_services/constants.py (hunk 1 above)
Comment (Herklos) [reply octobot_services/constants.py:68]: Did you forget to commit the fix ?

Comment (valouvaliavlo) [reply octobot_services/constants.py]: strange 🤔

Comment (valouvaliavlo) [reply octobot_services/constants.py]: now its' fix

**Review by GuillaumeDSM — APPROVED** [review]

File: octobot_services/constants.py (hunk 1 above)
Comment (Herklos) [reply octobot_services/constants.py]: Nice :+1:

**Comment by Herklos** [comment]
Can you also pump the package version by updating the last version number in : 
- `octobot_services/__init__.py`
-  the first line of `README.md`
And adding version details in `CHANGELOG.md` please ?

**Review by Herklos — APPROVED** [review]
:+1:

**Comment by Herklos** [comment]
Thanks @valouvaliavlo !

**Pull request merged**